### PR TITLE
Add a hover icon for external links.

### PR DESCRIPTION
**Pull request by floitsch**
This shows when links go to a different server.

### CODE REVIEW

**Comment by floitsch** [comment]
Here is a screenshot when the icon shows. (By default it's not visible, and only appears when hovering over links).
image

File: src/components/general/ExternalLink.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.external-container {
```
Comment (kasperl) [inline src/components/general/ExternalLink.css:1]: No copyright in these files?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/components/general/ExternalLink.css (hunk 1 above)
Comment (floitsch) [reply src/components/general/ExternalLink.css:1]: The majority of the files here don't have any copyright header.
Should probably get fixed...

**Pull request merged**